pages/licence-fees/index.js

```diff
@@ -0,0 +1,55 @@
+const { Router } = require('express');
+const routes = require('./routes');
+
+module.exports = () => {
+  const app = Router();
+
+  app.param('year', (req, res, next, year) => {
+    req.year = year;
+    next();
+  });
+
+  app.get('/', (req, res, next) => {
+    Promise.resolve()
+      .then(() => req.api(`/billing`))
+      .then(response => {
+        const year = response.json.meta.year;
+        res.redirect(req.buildRoute('fees.overview', { year }));
+      })
+      .catch(next);
+  });
+
+  app.use('/:year', (req, res, next) => {
+    const query = { year: req.year };
+    Promise.resolve()
+      .then(() => req.api('/billing', { query }))
+      .then(response => {
+        const startDate = response.json.meta.startDate;
+        const endDate = response.json.meta.endDate;
+        const numPils = response.json.data.numberOfPils;
+        const numPels = response.json.data.numberOfPels;
+        const fees = response.json.data.fees;
+        const personal = response.json.data.personal;
+        const establishment = response.json.data.establishment;
+        const total = response.json.data.total;
+
+        res.locals.static.fees = {
+          numPils,
+          numPels,
+          fees,
+          personal,
+          establishment,
+          total,
+          startDate,
+          endDate,
+          year: req.year
+        };
+      })
```

Comment:
```
.then({ json: { meta, data }} => {
  res.locals.static.fees = {
    ...pick(meta, ['startDate', 'endDate']),
    ...pick(data, ['fees', 'personal', 'establishment', 'total']),
    numPils: data.numberOfPils,
    numPels: data.numberOfPels,
    year: req.year
  };
})
```

Reply:
WHO ARE YOU AND WHAT HAVE YOU DONE WITH PHIL?!

Reply:
![](https://media.giphy.com/media/R3FUSQ5H5jzVe/giphy.gif)

Reply:
haha trying to fill in for @joefitter

Reply:
Too far. Too. Far.